formats.go

```diff
@@ -13,6 +13,8 @@ var (
 	nameRegex              = regexp.MustCompile(`^[a-zA-Z0-9][a-z0-9A-Z\. \-_]{2,128}$`)
 	codeRegex              = regexp.MustCompile("^([0-9abcdefghjkmnpqrtuvwxyz]{16}|[0-9]{6})$")
 	featureValueLabelRegex = regexp.MustCompile(`^[a-z0-9][a-z0-9-_\.]{1,128}$`)
+	annotationKey          = regexp.MustCompile(`^([^-\.\/][a-z0-9-\.\/]*[^-\.\/]|[a-z0-9-])$`)
+	annotationValue        = regexp.MustCompile(`^([^-\.\/][a-zA-Z0-9-\.\/]*[^-\.\/]|[a-z0-9-])$`)
```

Comment:
same problem with OR here I think?
Also the leading characters aren't very prescriptive, it essentially allows for any character thats not `- . /` at the start and end. This doesn't seem correct because that allows for many things to get in the start and finish, but not in the body.